### ISSUE TITLE
How to save the loss and acc of each batch  to the file? And how to output the results of each layer?

### ISSUE THREAD

**Issue by Imorton-zd** [fchollet/keras#571]
I am running the keras eample-imdb. I am a fresh man, leaning DL not long. I like the keras framework. It looks so easy. Besides the question of title, I also want to know how to save the output of each layer. Thanks!

**Comment by Imorton-zd** [comment]
P.S  I used the imdb_cnn. 
Expect more eamples and study materials about text classfication(or sentiment classification) by keras. Thanks!
